content/docs/tech-insights/define-custom-data-sources/index.md

```diff
@@ -134,10 +132,20 @@ After successfully adding a fact you will be able to select kind and type of ser
 
 ## Defining targeted entities
 
+You can define filters to target which entities should this data source consider to collect data from.
+In the Applies to section you can select which entites will be included in this data source and which entities will be excluded.
+
 ![Data Source Entity Filter](./data-source-entity-filter.png)
 
-You should be able to see the created Data Source in the overview screen. If you decide to create a draft Data Source, you will need to publish it in order for others to see it. This can be achieved using actions menu.
+The above configuration means the data source will collect data for all components with type library and service which don't have the infrastructure tag
+
+Providing multiple selection in a single input results in an `OR` relation between the selections.
+Selecting from multiple input fields end up with a relation `AND` between the fields.
 
+For example:
+Selecting `type: service, tags: infrastructure` will exclude every enetity Which has type service `AND` contains an infrastructure tag. However it won't exclude services which does not contain the tag, or components that contain only the tag but they are not services.
```

Comment:
`entity` rather than `enetity`

Reply:
`However it won't exclude services which do not contain the tag` rather than `However it won't exclude services which does not contain the tag`